Automattic/newspack-electionkit
# feat: adding ElectionKit profile custom post type and taxonomy

### All Submissions:







### Changes proposed in this Pull Request:

Adds ElectionKit Profile as a custom post type, and District and Race as a custom taxonomy.  Staging for future functionality.  The profiles are not publically accessible until we develop use cases for them.





### How to test the changes in this Pull Request:

1.  Install plugin, observe ElectionKit Profile custom post type.  Try to add a profile.
2.
3.

### Other information:
